deployment/linux/README.md

```diff
@@ -0,0 +1,174 @@
+# Kernel Configuration with `make menuconfig`
+You can find Elastic Defend's official list of supported Linux distributions and kernels here. 
+
+Outside this list, there may be cases where a Linux kernel does not provide all the capabilities required for Defend to run. The following are experimental and unsupported steps to configure a Linux kernel to run Elastic Defend.
```

Comment:
Since the instructions mention `emerge` perhaps mention they're examples for Gentoo?

```suggestion
Outside this list, there may be cases where a Linux kernel does not provide all the capabilities required for Defend to run. The following are experimental and unsupported steps to configure a Linux kernel on Gentoo to run Elastic Defend.
```

Reply:
Good call, changed.